build.gradle.kts

```diff
@@ -5,6 +5,7 @@ plugins {
     buildsrc.conventions.publishing
     buildsrc.conventions.`git-branch-publish`
     buildsrc.conventions.`yaml-testing`
+    id("dev.adamko.dokkatoo-html") version "2.2.0"
```

Comment:
Wasn't Dokkatoo merged into Dokka?

Reply:
Yes, but only as a formality, it hasn't been worked in. I'm still developing Dokkatoo in a separate repo.